addons/l10n_tr/data/account_tax_report_data.xml

```diff
@@ -91,6 +91,17 @@
                 <field name="name">Sales VAT</field>
                 <field name="hierarchy_level">0</field>
                 <field name="children_ids">
+                    <record id="sales_vat_line_0" model="account.report.line">
```

Comment:
You need to add `sequence` everywhere, wouhou! :smile: 
If you take a db in 17.0 without your commit, then update with yours, then this will be the last line of the report
Shorry for that